app/assets/stylesheets/items.scss

```diff
@@ -0,0 +1,63 @@
+.item {
+    font-size: 16px;
+    font-weight: bolder;
+    text-align: center;
+    margin-top: 15px;
+    z-index: 1;
+}
+
+.category {
+    font-size: 14px;
+    text-align: center;
+    color: gray;
+}
+
+.price {
+    color: $primary-color;
+    font-weight: bold;
+    text-align: center;
+}
+
+.menu-image {
+    width: 100%;
+    img {
+        border-radius: 50%;
+        width: 200px;
+        height: 200px;
+        overflow: hidden;
+        margin-bottom: 20px;
+        margin-top: 20px;
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+    }
+}
+.wrapper {
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 50px;
+    font-family: $font;
+    margin-bottom: 200px;
+}
+.row {
+    .col-lg-3, .col-md-4, .col-sm-6:not(:nth-of-type(4n+0)) {
+        border-right: solid;
+        border-width: 3px;
+        border-color: $secondary-color;
+    }
+    .col-lg-3, .col-md-4, .col-sm-6 {
+        border-bottom: solid;
+        border-width: 3px;
+        border-color: $secondary-color;
+        padding-bottom: 20px;
+        background-color: $body-bg;
+    }
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
Use bootstrap 4's predefined [media queries](https://getbootstrap.com/docs/4.0/migration/#responsive-utilities)

This could become 
```
  @include media-breakpoint-down(sm) {

  }
```